app/metrics.yaml

```diff
@@ -254,3 +260,249 @@ control:
       - fxr-telemetry@mozilla.com
       - dmu@mozilla.com
     expires: "2020-11-01"
+
+pages:
+  page_load:
+    type: timing_distribution
+    description: >
+      Measuring how long each page takes to load.
+    time_unit: nanosecond
+    bugs:
+      - https://github.com/MozillaReality/FirefoxReality/issues/2279
+    data_reviews:
+      - https://github.com/MozillaReality/FirefoxReality/pull/xxx
```

Comment:
Please don't forget to add the data review response link here and to the other metrics.